starter-code/index.css

```diff
@@ -0,0 +1,61 @@
+.blueBackground{
+  background-image: url(bgtile.png);
```

Comment:
You didn't need to put images in your root directory. You can access them with "images/bgtile.png" 😉 
/!\ And don't forget the quotes around "bgtile.png"